corehq/util/metrics/prometheus.py

```diff
@@ -86,11 +86,12 @@ def _get_metric(self, metric_type, name, tags, documentation, **kwargs):
             if getattr(settings, 'PUSHGATEWAY_HOST', None):
                 self._push_to_gateway()
             return metric.labels(**tags) if tags else metric
-        except ValueError:
+        except ValueError as err:
             prometheus_soft_assert(False, 'Prometheus metric error', {
                 'metric_name': name,
                 'tags': list(tags),
-                'expected_tags': metric._labelnames
+                'expected_tags': metric._labelnames,
+                'exception': err
```

Comment:
confirming this doesn't need to be cast to a string?

Reply:
it doesn't need to be cast here